api-metastore/src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -52,10 +52,11 @@ public SubscriptionController(final SubscriptionService subscriptionService) {
             @RequestParam(value = "limit", required = false, defaultValue = "20") final int limit,
             @RequestParam(value = "offset", required = false, defaultValue = "0") final int offset,
             @RequestParam(value = "token", required = false) final String token,
+            @RequestParam(value = "readers", required = false) final Set<String> readers,
```

Comment:
```suggestion
            @Nullable @RequestParam(value = "readers", required = false) final Set<String> readers,
```
See the declaration of `eventTypes` parameter a few lines up: I think we have the same requirements here.